### ISSUE TITLE
Video noise when playing games 

### ISSUE THREAD

**Issue by Leonprotos** [SmokeMonsterPacks/Mega-Sg-Jailbreak#57]
**Describe the bug**
Mega SG producing video noise on SMS games and some Sega CD games (Though it's way worse on SMS games) I get pixelated dots that flicker around the screen, and I've never had these before. Only happens with firmware above 4.4 and its jailbreak equivalents.

I have already tried different controllers, power, and HDMI cables and the problem persists. Furthermore I have tried using the games on real hardware and these issues do not present themselves.

**To Reproduce**
1. Toggle setting 'Issue happens regardless of settings'
2. Load ROM 'Games tested were Zillion and Rocky on SMS'
3. Play through 'Takes several minutes, but let the game run or play and inevitably the noise will appear, usually on red or blue fields.'

**Expected behavior**
After several minutes video noise will appear on SMS games, see examples below

**Screenshots**
![20190909_190824](https://user-images.githubusercontent.com/55200922/64711950-108ba380-d480-11e9-8fb0-c71fd8c22f49.jpg)
![20190909_181733](https://user-images.githubusercontent.com/55200922/64711956-13869400-d480-11e9-8884-b56c8058fd3e.jpg)
![20190911_084925](https://user-images.githubusercontent.com/55200922/64711963-15e8ee00-d480-11e9-9167-69427a3416ba.gif)

**Firmware version**
Official firmware 4.6 or JB7.6, issue happens on both.

**Relevant Mega Sg settings**
Issue happens regardless of settings, but these are the values used:
Resolution:  1080P
horizontal pos=10, 
vertical pos=45
H Value: 1462
V Value: 1200 
enable H interpolation
disable V interpolation

All other settings default

**ROM name and hash values**
Zillion (US)
CRC32	5718762C
MD5	D4BF9E7BCF9A48DA53785D2AE7BC4270
SHA-1	3AAB3E95DAC0FA93612DA20CF525DBA4DC4CA6BA

Rocky (World)
CRC32	1BCC7BE3
MD5	582D58D95C9774CD50555366C253A050
SHA-1	8601927CA17419F5D61F757E9371CE533228F7BB

The issue happens with real cartridges, Everdrive X5, Mega SD, and loading via SD Card on JB Firmware.

**Comment by Rageypoo** [comment]
will verify later tonight

**Comment by skooterblog** [comment]
I'm using JB 7.6. I loaded Zillion from the SD Card and let it running for ~2 hours. Then I did the same with Rocky. I couldn't reproduce this issue.

**Comment by Rageypoo** [comment]
I too was unable to verify on both 7.5 and 7.6jb with both games. I also asked for the original ROMS used just in case. After discussing further we found that the problem was able to be reproduced by messing with scanlines with version 7.6. My system was not affected by the scanline issues however, and I was unable to reproduce after following instructions.

**Comment by frederic-mahe** [comment]
@Rageypoo @skooterblog thank you for following up on that bug report.

I am going to close that issue but feel free to comment if someone finds a sure way to replicate it.

**Comment by TheRealFlamepanther** [comment]
The *.6 firmwares introduced this issue for me too. However, it isn't only SMS games that are affected. Genesis games have the same issue, but it's harder to see. The noise is not so obvious as the examples make it look, because each individual miscolored pixel is present for about 1 frame. In action, it looks like color noise you might see in analog video formats. Mostly purple-pink spots in blue areas, sometimes red-pink spots in red areas. Issue goes away if I roll back to an older firmware.

I heard through the grapevine that Kevtris is aware of the issue and has a possible fix already, but I still thought I should pipe up that this is an actual issue that exists.

**Comment by SegaSnatcher** [comment]
I feel this issue should be reopened based on this user's post.

https://twitter.com/vpp/status/1182270128995614722

**Comment by RowanUnderwood** [comment]
What TV are you connected to?  I have the same issue on a oled55c9pua.

**Comment by frederic-mahe** [comment]
@RowanUnderwood you confirm that you've observed these glitches when playing non-SMS games? I am going to modify the title of the issue to reflect that.

**Comment by RowanUnderwood** [comment]
Yes, all of the Genesis (Megadrive) games I tested also had the issue.

**Comment by Leonprotos** [comment]
> What TV are you connected to? I have the same issue on a oled55c9pua.

I'm using an LG OLED55C6P

**Comment by TheRealFlamepanther** [comment]
> What TV are you connected to? I have the same issue on a oled55c9pua.

Vizio E420d-A0

**Comment by thomas83lin** [comment]
I actually just started getting this issue.

I had just updated from 7.5jb to 7.6jb and switched to a different tv. So I'm not sure if this problem was present on 7.5jb or if the tv is causing it. 

I started noticing this while using the Mega SG but once trying to isolate the issue I noticed I'm still getting the flickering pixels even while booting straight to the Mega SG menu selection so I getting them everywhere even with no cart connected or before playing a game.

TV model Samsung UN32J5003AF
update:
Figured I'd do minimum testing before posting 
Switching back TV's didn't fix it.
Downgrading back to 7.5jb seemed to fix the problem!!
